docs/reference/commandline/image_history.md

```diff
@@ -76,3 +77,43 @@ $ docker history --format "{{.ID}}: {{.CreatedSince}}" busybox
 f6e427c148a7: 4 weeks ago
 <missing>: 4 weeks ago
 ```
+
+### <a name="platform"></a> Show history for a specific platform (--platform)
+
+The `--platform` option allows you to specify which platform-variant to show
+history for if multiple platforms are present. By default, `docker history`
+shows the history for the daemon's native platform or if not present, the
+first available platform.
+
+If the local image store has multiple platform-variants of an image, the
+`--platform` option selects which variant to show the history for. An error
+is produced if the given platform is not present in the local image cache.
+
+The platform option takes the `os[/arch[/variant]]` format, for example,
```

Comment:
nit
```suggestion
The platform option takes the `os[/arch[/variant]]` format, for example
```

Reply:
Let me check with the grammar police (@dvdksn 🙏 ), but I think  it's recommended to always follow `for example` with a comma. Perhaps it should start with a new sentence here though (`format. For example, ...`)

Reply:
the word man! anyway these were just my 2 cents, so feel free to merge whenever 😄

Reply:
If the "For example" starts the sentence, I agree with having the comma after it. As is, when reading the sentence it feel strange to pause both before and after "for example", because it makes it sound like the example came before (causing some confusion, at least to myself 😄).

Definitely minutia though

Reply:
Yeah; these are hard. I basically tried to avoid ending up with "3 word sentences" because that also felt a bit silly (which is why I used the comma before `example`). I updated to use a semicolon instead, but not sure if that's 100% grammatically correct either 🥹